From 29df93de98a45c17513f68d877ef3435aa9fbc11 Mon Sep 17 00:00:00 2001
From: Anson Huang <b20788@freescale.com>
Date: Sat, 19 Nov 2011 10:43:14 +0800
Subject: [PATCH 0638/2463] ENGR00162646 [DVFS]Fix build warning

Fix dvfs_core build warning:

  arch/arm/plat-mxc/dvfs_core.c:824: warning: 'downthreshold_show'
  defined but not used

  arch/arm/plat-mxc/dvfs_core.c:830: warning: 'downthreshold_store'
  defined but not used

  arch/arm/plat-mxc/dvfs_core.c:842: warning: 'downcount_show' defined
  but not used

  arch/arm/plat-mxc/dvfs_core.c:848: warning: 'downcount_store' defined
  but not used

Signed-off-by: Anson Huang <b20788@freescale.com>
---
 arch/arm/plat-mxc/dvfs_core.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/plat-mxc/dvfs_core.c b/arch/arm/plat-mxc/dvfs_core.c
index f1a74ba..0065304 100755
--- a/arch/arm/plat-mxc/dvfs_core.c
+++ b/arch/arm/plat-mxc/dvfs_core.c
@@ -820,7 +820,7 @@ void dump_dvfs_core_regs()
 			__raw_readl(dvfs_data->membase
 				    + MXC_DVFSCORE_THRS + 0x40));
 }
-
+#if 0
 static ssize_t downthreshold_show(struct device *dev,
 				struct device_attribute *attr, char *buf)
 {
@@ -856,7 +856,7 @@ static ssize_t downcount_store(struct device *dev,
 
 	return size;
 }
-
+#endif
 
 static ssize_t dvfs_enable_show(struct device *dev,
 				struct device_attribute *attr, char *buf)
-- 
1.7.7.4

